docs/source/tutorials/quickstart.rst

```diff
@@ -2,145 +2,150 @@
 Quickstart
 ==========
 
-This document will lead you step by step through the necessary procedure to get
-started with manim for the first time as soon as possible.  This tutorial
-assumes you have already installed manim following the steps in
-:doc:`../installation`.
+.. note::
+ Before proceeding, install manim and make sure it's running properly by
+ following the steps in :doc:`../installation`. For
+ information on using manim with Jupyterlab or Jupyter notebook, go to the
+ documentation for the
+ :meth:`IPython magic command <manim.utils.ipython_magic.ManimMagic.manim>`,
+ ``%%manim``.
 
+Overview
+*************************************************
 
-Start a new project
-*******************
+This quickstart guide will lead you through creating a sample project using manim: an animation
+engine for precise programmatic animations.
 
-To start a new manim video project, all you need to do is choose a single
-folder where all of the files related to the video will reside.  For this
-example, this folder will be called ``project``,
+First, you will use a command line
+interface to create a ``Scene``, the class through which manim generates videos.
+In the ``Scene`` you will animate a circle. Then you will add another ``Scene`` showing
+a square transforming into a circle. This will be your introduction to manim's animation ability.
+Afterwards, you will position multiple manim objects (mobjects). Finally, you
```

Comment:
```suggestion
Afterwards, you will position multiple mathematical objects (mobjects). Finally, you
```
_Technically_, I think `Mobject` stands for mathematical object.

Reply:
Noted. Is "mobject" supposed to be capitalized? Looking at other docs, it seems like it's mobject when talking about them in aggregate and then `Mobject` when talking about the class? I saw that you suggested the capital M a few times later on in your comments but didn't in others so I wanted some clarification there.

Reply:
I think it should be `Mobject`, but since there isn't a style guide yet I think you can just use what you prefer. It'll hopefully eventually get standardized.